test/chain-test.js

```diff
@@ -815,6 +815,36 @@ describe('Chain', function() {
       'bad-txns-txouttotal-toolarge');
   });
 
+  it('should fail to connect total fee toolarge', async () => {
+    const job = await cpu.createJob();
+
+    Selector.MAX_FEE = 50 * consensus.COIN;
+
+    const outputs = [{ address: wallet.getAddress(), value: 0 }];
+    const tx1 = await wallet.send({
+      outputs: outputs,
+      hardFee: Selector.MAX_FEE
+    });
+    job.pushTX(tx1.toTX());
+
+    const tx2 = await wallet.send({
+      outputs: outputs,
+      hardFee: Selector.MAX_FEE
+    });
+    job.pushTX(tx2.toTX());
+
+    consensus.MAX_MONEY = tx1.getFee() + tx2.getFee() - 1;
+
+    try {
```

Comment:
This try should probably wrap everything above, after Selector.MAX_FEE and consensus.MAX_MONEY have been changed. 

And it may be worth keeping a copy of the original value, while it's not likely to change in the future, it's possible that MAX_FEE could, and the restored value would be incorrect.

Reply:
Sounds good. I moved the only lines which don't depend on the swapped values out, so they don't obscure any failures.